src/java/Test.java

```diff
@@ -159,6 +160,27 @@ public static void testRemoveDuplicatesUnsortedList() {
         singlyLinkedList.removeAll();
     }
 
+    public static void testArrayBubbleSortAscending(){
+        SortingAlgorithms sortingAlgorithms = new SortingAlgorithms();
+        int[] array = {6,3,7,2,10,9};
+        System.out.println("Array before method call: ");
+        sortingAlgorithms.display(array);
+        int[] sortedArray = sortingAlgorithms.arrayBubbleSortAscending(array);
+        System.out.println("Array after method call: ");
+        sortingAlgorithms.display(sortedArray);
+        System.out.println("---------------------------------------");
+        System.out.println("When all items are the same: ");
+        array = new int[]{2,2,2,2,2};
+        sortingAlgorithms.arrayBubbleSortAscending(array);
+        sortingAlgorithms.display(array);
+        System.out.println("---------------------------------------");
+        System.out.println("When array is empty: ");
+        array = new int[]{};
+        sortingAlgorithms.arrayBubbleSortAscending(array);
+        sortingAlgorithms.display(array);
+
+    }
+
```

Comment:
This is not part of this task. You probably branched from wrong branch (bubble_sort).